front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts

```diff
@@ -141,6 +145,38 @@ export abstract class TransactionTypeBaseComponent implements OnInit, OnDestroy
     this.contactId$.complete();
   }
 
+  retrieveMemoText(formValues: any) {
```

Comment:
@Elaine-Krauss-TCG This method seems to be working but makes me uncomfortable because it calls the transactionType object directly from the "this" object which means it is possibly not calling the childTransactionType when appropriate for screens like the EARMARK RECEIPT page with 2 transactions, a parent and child.

I'm thinking it safer to pass the transactionType in the method parameters: retrieveMemoText(formvalues: any, transactionType: TransactionType) and then replace the this.transactionType. with just transactionType. within the body of the method.

Reply:
That's a really good call.  Done.